src/eduid_userdb/userdb.py

```diff
@@ -288,3 +288,19 @@ def update_user(self, obj_id, attributes):
                         '$set': attributes,
                     }
                 )
+
+    def get_identity_proofing(self, user):
+        """
```

Comment:
And in any case I *did* look into it, and added user.nins.count where I should have used verified :-(